lib/Client.js

```diff
@@ -1667,6 +1674,10 @@ class Client extends EventEmitter {
     * @arg {Boolean} [content.allowedMentions.everyone] Whether or not to allow @everyone/@here.
     * @arg {Boolean | Array<String>} [content.allowedMentions.roles] Whether or not to allow all role mentions, or an array of specific role mentions to allow.
     * @arg {Boolean | Array<String>} [content.allowedMentions.users] Whether or not to allow all user mentions, or an array of specific user mentions to allow.
+    * @arg {Array<Object>} [content.attachments] Attachment objects to retain and/or edit the name and/or description for currently attached files and add the filename and description for new files
+    * @arg {String} [content.attachments[].description] The file description
+    * @arg {String} [content.attachments[].filename] The file's name. This is not needed if you are attaching a new file
+    * @arg {Number} content.attachments[].id The file's ID. If you are attaching a new file, this would be the index ID of the attaching file
```

Comment:
```suggestion
    * @arg {(Number | String)} content.attachments[].id The file's ID. If you are attaching a new file, this would be the index ID of the attaching file
```